frontend/packages/data-portal/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:20.8-alpine as build
+
+COPY . /app
```

Comment:
Just curious, do we need files related to linting to be a part of the build, or could we add those to the `.dockerignore`?